bHY6yh-d42838/cod-bqe7s7_bHY6yh-d42838.xml

```diff
@@ -0,0 +1,407 @@
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e554">
+               <ref xml:id="bHY6yh-d42838-Rd1e613">Idem capitulo iiii</ref>
+        <quote xml:id="bHY6yh-d42838-Qd1e556">Iudicis auxilium: sub iniquo teste rogato.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e562">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e626">Cassiodorus epistolarum libro primo epistola iii</ref> <quote xml:id="bHY6yh-d42838-Qd1e566">fa<lb ed="#P" break="no"/>cilis 
+               recti persuasor est innocens iu<lb ed="#P" break="no"/>dex
+               excessus tunc sunt in formidine cum 
+               <lb ed="#P"/>creduntur iudicibus displicere.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e577"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e640">Ibidem libro iii epistola xxviii</ref> <quote xml:id="bHY6yh-d42838-Qd1e579">tamdium iudex 
+               <lb ed="#P"/>dicitur quamdiu iustus putatur.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e585">
+               OMITTED
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e593"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e661">Ibidem xxviiii epistola</ref> <quote xml:id="bHY6yh-d42838-Qd1e595">inumbrat famam
+               <lb ed="#P"/>publicam in consule tenacitatis obscuritas.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e601">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e673">Ibidem libro iiii epistola x</ref> <quote xml:id="bHY6yh-d42838-Qd1e605">Quid a belli<lb ed="#P" break="no"/>ca 
+               confusione pax tranquilla distabit si 
+               <lb ed="#P"/>per vim litigia terminentur.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e613"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e685">Ibidem xx epistola</ref> <quote xml:id="bHY6yh-d42838-Qd1e615">dignus est inter alios 
+               <lb ed="#P"/>sequenda decernere qui sibi modum isuis 
+               <lb ed="#P"/>scit legittime conuersationis imponere.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e623"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e697">Ibidem libro vi epistola iiii</ref> <quote xml:id="bHY6yh-d42838-Qd1e625">Est nimirum 
+               <lb ed="#P"/>magnum / et singulare praeconium si iudi<lb ed="#P" break="no"/>ces 
+               non accipiant vbi sunt / qui multum da<lb ed="#P" break="no"/>re 
+               contendunt.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e636"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e712">Ibidem xxi epistola</ref> <quote xml:id="bHY6yh-d42838-Qd1e638">Prima equitas est a se 
+               <lb ed="#P"/>consules inchoare ut vel criminosi  
+               <lb ed="#P"/>nequeunt similem reperire nullas enim in se 
+               <lb ed="#P"/>culpas timet reus quas habere conspe<lb ed="#P" break="no"/>xerut 
```

Comment:
something seems wrong with "conspexerut" should probably be "conspexerunt" or "conspexerit". If it should stay as is, then it should be in a `<sic>` tag
```suggestion
               <lb ed="#P"/>culpas timet reus quas habere conspe<lb ed="#P" break="no"/>xerunt 
```